stripe/res/layout/activity_payment_sheet.xml

```diff
@@ -5,76 +5,74 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <LinearLayout
         android:id="@+id/bottom_sheet"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:clipToPadding="true"
+        android:orientation="vertical"
```

Comment:
Was there an issue with `clipToPadding`?

Reply:
I didn't see why it was needed, and was going to ask you why it was added. Was it fixing something?

Reply:
Actually `clipToPadding` defaults to true so this is a no-op.